### PR TITLE
get label map from predictor

### PR DESCRIPTION
**Pull request by wzhongyuan**
A simple change to get label map from predictor instead of reading it  separately

Verified by the example itself

### CODE REVIEW

File: models/src/main/python/bigdl/zoo/example/ImageClassificationPredict.py [hunk 1]
```diff
@@ -32,10 +32,10 @@ def predict(modelPath, imgPath, topN):
     print ("Model Path %s" % modelPath)
     print ("Image Path %s" % imgPath)
     print  ("Top N : %d" % topN)
-    labelMap = read_imagenet_label_map()
     model = Model.loadModel(modelPath)
     imageFrame = ImageFrame.read(imgPath)
     predictor = Predictor(model)
+    labelMap = predictor.configure.label_map()
```
Comment (jason-dai) [inline models/src/main/python/bigdl/zoo/example/ImageClassificationPredict.py]: Shall we add label_map() method to Predictor? @SeaOfOcean

File: models/src/main/scala/com/intel/analytics/zoo/models/Predictor.scala [hunk 2]
```diff
@@ -89,6 +89,14 @@ class Predictor[T: ClassTag](
     // apply post process if defined
     if (null != configure.postProcessor) configure.postProcessor(result) else result
   }
+
+  /**
+   * get label map for predictor
+   */
+
```
Comment (SeaOfOcean) [inline models/src/main/scala/com/intel/analytics/zoo/models/Predictor.scala]: remove this empty line

**Comment by jason-dai** [comment]
Please update the scala examples as well.

**Comment by wzhongyuan** [comment]
@jason-dai   scala example updated as well

**Comment by jason-dai** [comment]
LGTM

**Pull request merged**